apps/perf-test-react-components/package.json

```diff
@@ -19,14 +19,14 @@
   },
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28426&commitId=af930bb94deae40b78189e2d58b0bbe75fa7caa0&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Compact.default.chromium.png|27| Changed|
|ActivityItem.Root - RTL.default.chromium.png|695| Changed|
|ActivityItem.Root.default.chromium.png|1083| Changed|

</details>


  <details><summary><b>Breadcrumb</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb.Root - RTL.itemLinkFocus.chromium.png|7211| Changed|
|Breadcrumb.Root.default.chromium.png|7065| Changed|
|Breadcrumb.Root.itemLinkFocus.chromium.png|7416| Changed|
|Breadcrumb.Root.overflowButtonFocus.chromium.png|7118| Changed|
|Breadcrumb.Root - RTL.click.chromium.png|7334| Changed|
|Breadcrumb.Root - RTL.overflowButtonFocus.chromium.png|6908| Changed|
|Breadcrumb.Root.longTitleHover.chromium.png|7253| Changed|
|Breadcrumb.Root - RTL.longTitleHover.chromium.png|7029| Changed|
|Breadcrumb.Root - RTL.default.chromium.png|6862| Changed|
|Breadcrumb.Root - RTL.shortTitleHover.chromium.png|6938| Changed|
|Breadcrumb.Root.click.chromium.png|7084| Changed|
|Breadcrumb.Root - RTL.hover.chromium.png|7119| Changed|
|Breadcrumb.Root.shortTitleHover.chromium.png|7199| Changed|
|Breadcrumb.Root.hover.chromium.png|7337| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>